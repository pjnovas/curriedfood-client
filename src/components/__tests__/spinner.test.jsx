import Spinner from '../spinner';

takeSnapshots(
  {
    default: {},
    asBar: {
      bar: true
    }
  },
  Spinner
);
