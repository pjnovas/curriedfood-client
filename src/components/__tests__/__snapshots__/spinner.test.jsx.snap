// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`asBar 1`] = `
<withTheme(ProgressBar)
  animating={true}
  color="#ad1457"
  indeterminate={true}
/>
`;

exports[`default 1`] = `
<withTheme(ActivityIndicator)
  animating={true}
  color="#ad1457"
/>
`;
